config.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"github.com/mitchellh/go-homedir"
```

Comment:
The Makefile uses `.build/` with the exact same purpose, so yeah, we should rename it probably.